### PR TITLE
Feature/cr 3114 update origin func

### PR DESCRIPTION
**Pull request by idan-aviv**

### CODE REVIEW

File: cloudrail/knowledge/context/mergeable.py [hunk 1]
```diff
@@ -13,6 +14,12 @@ class EntityOrigin(str, Enum):
     PSEUDO = 'pseudo'
 
 
+class UnSupportedEntityOriginType(Exception):
```
Comment (Tomerva) [inline cloudrail/knowledge/context/mergeable.py:17]: rename to UnSupportedEntityOriginTypeException

File: cloudrail/knowledge/context/mergeable.py [hunk 2]
```diff
@@ -13,6 +14,12 @@ class EntityOrigin(str, Enum):
...
+
+    def __init__(self) -> None:
+        super().__init__('origin entity type don\'t supported')
```
Comment (Tomerva) [inline cloudrail/knowledge/context/mergeable.py:20]: change to 'IaC type {iac_type} not supported' and pass the iac_type = self.iac_state.iac_type

**Review by Tomerva — CHANGES_REQUESTED** [review]

File: cloudrail/knowledge/context/mergeable.py (hunk 2 above)
Comment (idan-aviv) [reply cloudrail/knowledge/context/mergeable.py:21]: @Tomerva the logic cause to exception is on origin entity not iac type

Comment (Tomerva) [reply cloudrail/knowledge/context/mergeable.py:21]: @idan-aviv but the error is originated from the value of self.iac_state.iac_type

File: cloudrail/knowledge/context/mergeable.py [hunk 3]
```diff
@@ -5,12 +5,20 @@
...
 
 
 class EntityOrigin(str, Enum):
     LIVE_ENV = 'live_environment'
     TERRAFORM = 'terraform'
     PSEUDO = 'pseudo'
+    CLOUDFORMATION = 'cloudformation'
+
+
+class UnSupportedIacType(Exception):
```
Comment (Tomerva) [inline cloudrail/knowledge/context/mergeable.py:18]: rename to UnSupportedIacTypeException (all our exceptions end with Exception)

File: cloudrail/knowledge/context/mergeable.py [hunk 4]
```diff
@@ -5,12 +5,20 @@
...
+
+    def __init__(self) -> None:
+        super().__init__('IaC type don\'t supported')
```
Comment (Tomerva) [inline cloudrail/knowledge/context/mergeable.py:21]: IaC type not supported

**Review by Tomerva — CHANGES_REQUESTED** [review]

**Review by Tomerva — APPROVED** [review]

**Review by Tomerva — APPROVED** [review]

**Pull request merged**